spring-cloud-generator/generate-one.sh

```diff
@@ -44,6 +48,20 @@ if [[ $download_repos -eq 1 ]]; then
 fi
 
 cd googleapis
+git reset --hard $googleapis_commitish
+
+# In googleapis/WORKSPACE, find http_archive() rule with name = "gapic_generator_java",
```

Comment:
Do you think it makes sense for readability to extract these replacement logic into a separate script?

Reply:
Sure, will use a separate file